collectors/sentinelone/README.md

```diff
@@ -17,13 +17,7 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 ![ScreenShot](./docs/sentinelone_credentials.jpg)
 
-### 2. API Docs
-
-For API docs, You need to login your Management Console. After login, you can access the following links.
-1. [Authentication](https://usea1-swprd1.sentinelone.net/apidoc/#!/Accounts/get_web_api_v2_0_private_accounts_name_available)
-2. [Activities](https://usea1-swprd1.sentinelone.net/apidoc/#!/Activities/get_web_api_v2_0_activities) 
-
-### 3. CloudFormation Template (CFT) 
```

Comment:
can we have new links?

Reply:
The links are direct to a sandbox and require log-ins, so are irrelevant to anyone without access to our specific S1 sandbox.